code/modules/mob/living/carbon/human/say.dm

```diff
@@ -123,27 +129,14 @@
 		if("whisper")
 			whisper_say(message, speaking, alt_name)
 			return
-/*		if("binary")
-			if(robot_talk_understand || binarycheck())
-				robot_talk(message)
-			return
-		if("changeling")
-			if(mind && mind.changeling)
-				for(var/mob/Changeling in mob_list)
-					if((Changeling.mind && Changeling.mind.changeling) || istype(Changeling, /mob/dead/observer))
-						Changeling << "<i><font color=#800080><b>[mind.changeling.changelingID]:</b> [message]</font></i>"
-			return
-*/
 		else
-			if(!(stunned >= 4))
-				if(message_mode)
-					if(message_mode in (radiochannels | "department" | "special"))
-						if(l_ear && istype(l_ear,/obj/item/device/radio))
-							l_ear.talk_into(src,message, message_mode, verb, speaking)
-							used_radios += l_ear
-						else if(r_ear && istype(r_ear,/obj/item/device/radio))
-							r_ear.talk_into(src,message, message_mode, verb, speaking)
-							used_radios += r_ear
+			if(message_mode)
```

Comment:
Ah, missed that one. Re-adding it now.

There should be a more elegant way to implement "No radios while stunned" than having an if statement in each switch case though. Hmm